pkg/cluster/cluster.go

```
@@ -212,13 +212,17 @@ func (c *Context) provisionControlPlane(
 
 	// create kubeadm config file
 	kubeadmConfig, err := c.createKubeadmConfig(
```

We ignored this error previously, right?

---

yes, that was bad